stylesheet.css

```diff
@@ -0,0 +1,26 @@
+body {
+	background-color: #d3e0e4;
+}	
+
+h1 {
+	font-family: Trebuchet MS;
```

Comment:
If we set the font-family on the `body`, all the other elements on the page will inherit the same font, so we won't have to redeclare it lots of times.

Because Trebuchet MS is not a standard web font, we need to surround its name in quotes (`"Trebuchet MS"`). Also, font-family allows us to provide some fallbacks in case users don't have our first choice installed. So this could instead be: `font-family: "Trebuchet MS", Helvetica, sans-serif;`